demisto_sdk/commands/common/hook_validations/base_validator.py

```diff
@@ -61,3 +66,37 @@ def handle_error(self, error_massage, error_code, file_path, should_print=True,
                 click.secho(formatted_error, fg="bright_red")
 
         return formatted_error
+
+    def check_deprecated(self, file_path):
+        if file_path.endswith('.yml'):
+            yml_dict = get_yaml(file_path)
+            print(yml_dict)
+            if 'deprecated' in yml_dict and yml_dict['deprecated'] is True:
```

Comment:
need to check that this is indeed the case for PB, maybe it's `hidden` there

Reply:
Actually I don't see any `deprecated` flags on playbooks. This will only work on integrations and scripts unless we add one.

Reply:
So we should check the hidded one i think, @anara123 what do you think?

Reply:
should it be like `{hidden: {deprecated: true}}`?

Reply:
just {hidden: true}